osbenchmark/builder/models/config_instance_types.py

```diff
@@ -0,0 +1,10 @@
+from enum import Enum
+
+
```

Comment:
I think it's worth adding doc string to this Enum explaining its usage and significance

Reply:
Will do